### DISCUSSION TITLE
Can it be used with Livewire's new JS feature?

### DISCUSSION THREAD

**Discussion by sbc640964** [filamentphp/filament#7802] [Help]
[#Js Livewire Attribute](https://livewire.laravel.com/docs/actions#livewires-hybrid-javascript-functions)
I want to create an action in the form with js (i using with filament panels)

**Comment by danharrin** [comment]
What are you actually trying to achieve with the JS feature

**Reply by sbc640964** [reply]
@danharrin
In my specific case it's like this:
I have a credit debit iframe which I insert into a form with a View Field
According to the provider's instructions, I should stream to the iframe by a data event and send the iframe, actually, I don't send my form in the usual way, I want the submit button to actually access the form data with js and send to the iframe.

This is my specific case, but I believe there are many things it would have helped (js access to form or table data to interact with elements on the page).

**Reply by sbc640964** [reply]
@danharrin 
In my pink dreams I saw this: :)
```php
Forms\Components\Actions\Action::make('JsAction')
   ->actionJs(<<<'JS'
   alert('wow!!!');
   if(...){
     notification.send('success', 'GOOD!')
   }
JS;
)
```
